modules/vlybyBidAdapter.js

```diff
@@ -0,0 +1,76 @@
+import { registerBidder } from '../src/adapters/bidderFactory.js'
+
+import { BANNER, VIDEO } from '../src/mediaTypes.js'
+import { config } from '../src/config.js';
+
+const ENDPOINT = '//europe-west3-vlybypoc2019.cloudfunctions.net/prebid';
+const BIDDER_CODE = 'vlyby';
+
+export const spec = {
+  code: BIDDER_CODE,
+  supportedMediaTypes: [VIDEO, BANNER],
```

Comment:
Why banner if this is an outstream implementation?

Reply:
In general this Adapter provides a outstream Mediation for Publishers. They can use this Adapter in AdServer with a price Point from Endpoint. In some cases Publisher can use Banner. This makes this adapter more flexibel, for examaple to add Banner with cheaper eCPM for Fallback etc.